code/Forms/GridFieldDetailFormPreviewExtension.php

```diff
@@ -35,7 +35,6 @@ private function getSilverStripeNavigator(): DBHTMLText
     {
         $navigator = SilverStripeNavigator::create($this->owner->getRecord());
         $templates = SSViewer::get_templates_by_class(LeftAndMain::class, '_SilverStripeNavigator', LeftAndMain::class);
-        $renderWith = SSViewer::chooseTemplate($templates);
-        return $navigator->renderWith($renderWith);
+        return $navigator->renderWith($templates);
```

Comment:
This gives the _exact_ same result. There was no need for the `chooseTemplate()` there inbetween (which is good 'cause we can't do that anymore)